acceptance/auth/bundle_and_profile/.databrickscfg

```diff
@@ -0,0 +1,7 @@
+[DEFAULT]
+host = $DATABRICKS_HOST
+token = default
+
+[profile_name]
+host = https://test@bar.com
```

Comment:
Using real domain might result in sending auth tokens to it. Can you use something like https://test@non-existing-subdomain.databricks.com